### ISSUE TITLE
[Low] Consider using the post-pandemic numbers to further calibrate the model

### ISSUE THREAD

**Issue by Feynstein** [mrc-ide/covid-sim#194]
That one is is s bit out of my field, but I'll try at an idea anyway... Ok so I haven't had time to completely understand the code yet, or the model. But it seems to me that it takes into account a lot of variables, like houses and stuff. In my opinion this level of detail is bound to generate good results if it's applied correctly with the proper code backing it. What I suggest is that when all this stupid nonsense is done (the pandemic) you take a good look at all the parameters and you try to replicate the actual numbers that happened in the world. I don't really know yet how it works, but it seems to me that those parameters (maybe some of the defined ones) need to be re-optimized against the real data, once some degree of repeatability is obtained. And I think I know just how to do it. Non linear optimization using a cool library I know called NLOPT.

**Comment by weshinsley** [comment]
I imagine there will be decades of research to come after the pandemic when more data is available. This is really what epidemiology research is. Right now we are in the emergency stage where model runs are repeated multiple times in the week with calibrations to the latest data as it comes.

**Comment by Feynstein** [comment]
Yes and that's exactly why I intend to refactor the shit out of your code in the next week. I'm not like the other guys, when I see something that important that obviously needs help I take action. Especially if it's scientific development, which is exactly what I do on a daily basis.
